services/jobrunner/justfile

```diff
@@ -1,44 +1,48 @@
-jobrunner_cli := 'docker compose run --rm jobrunner python3 -m jobrunner.cli'
+jobrunner_cli := 'docker compose run --rm controller python3 -m jobrunner.cli'
 
 [private]
 default:
   @just --list
 
 
-# show the status of the jobrunner service
+# show the status of the jobrunner services
 status:
-    docker compose ps jobrunner
+    docker compose ps agent controller
 
-# stop the jobrunner service
+# stop the jobrunner services
 stop:
     docker compose stop
 
-# restart the jobrunner service
+# restart the jobrunner services
 restart *args:
-    docker compose up --no-build {{ args }} --detach jobrunner
+    docker compose up --no-build {{ args }} --detach agent controller
 
 alias start:=restart
 
 restart-if-running:
     #!/usr/bin/env bash
     set -euo pipefail
 
+    # Note that we check if the agent is running and, if so, restart the agent
+    # *and* the controller. This is a bit awkward but probably OK given that
+    # this state of affairs is supposed to be temporary.
+
     # The quadruple opening braces are not a typo: they are how we have to escape the braces
     # inside a Justfile (the closing braces do not need escaping)
-    JOBRUNNER_STATE=`docker compose ps --format "{{{{.State}}" jobrunner`
+    JOBRUNNER_STATE=`docker compose ps --format "{{{{.State}}" agent`
     if [ "$JOBRUNNER_STATE" = "running" ]; then
-        docker compose up --no-build --detach jobrunner
+        docker compose up --no-build --detach agent controller
     fi
 
-# Update jobrunner and restart
+# Update jobrunner services and restart
 deploy: && restart-if-running
-    docker compose pull --quiet jobrunner
+    docker compose pull --quiet agent controller
 
-# Run a jobrunner cli command
+# Run a controller cli command
```

Comment:
This could be a controller or an agent command (while they're both in the same place). 
`{{ jobrunner_cli }}` is `jobrunner.cli`; if `command` could be e.g. `agent.kill_task` or `controller.add_job`

Reply:
No, it can only be controller commands because it only runs against the controller container. If we need to run agent CLI commands then we'll need a separate Just command.